Entities/Ingame/Club.cs

```diff
@@ -48,13 +48,13 @@ public Club (int memoryAddress, ArraySegment<byte> originalBytes, IVersion versi
 
 		public string Name { 
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.Name, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.Name, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
 		public string ShortName {
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.ShortName, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.ShortName, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
```

Comment:
Can you please explain why you need to offset the Name by -0x1? This breaks compatibility with all other versions